### ISSUE TITLE
Clearing IndexedDB is not always enough

### ISSUE THREAD

**Issue by NadaAfterwards** [joaovitorbf/ynodesktop#18]
During version 1.1.7 of the client the fangame Uneven Dream updated to version 0.8. This version introduced a lot of changes to asset names and directories, primarily with the bedrooms of the two playable characters. When the UD update dropped, the client would fail to load the new assets making it too frustrating to progress.

I figured this was something that needed to be addressed in the next client's update since the option "Clear IndexedDB" did nothing to solve this. However when version 1.1.8 came out the problem persisted, I opted to uninstall and wipe anything left behind in AppData\Roaming\ynodesktop -- this finally solved the problem after a fresh install.

In short, it would be nice to have an extra option to clear whatever cache is used for this type of data as the problem might be recurring again right now with the latest version of Muma | Rope. I have not tried if this is the case yet.

Additionally I think the text "IndexedDB" is too technical to be understood by the average user. Even with the two warnings stating what data will be lost, that might not be inclusive of everything. I'm unsure how it can be better labeled in the context menu though...
